cost-analyzer/templates/aggregator-service.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostAggregator) }}
+{{- if gt (.Values.kubecostAggregator.replicas | toString | atoi) 0 }}
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "aggregator.serviceName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "aggregator.commonLabels" . | nindent 4 }}
+spec:
+  selector:
+{{ include "aggregator.selectorLabels" . | nindent 4 }}
+  type: "ClusterIP"
```

Comment:
StatefulSets require a headless service currently and it's not reflected here.

Reply:
wouldn't it be OK to not have a headless service in this scenario? As I understand it, headless services are useful if you want to address a specific member of the set. If this statefulset is scaled, we don't really care who answers a given request. We are sort of ignoring the stable network ID feature of the statefulset, focusing more on the persistence features

Reply:
If that's the case, why does this need to be a StatefulSet just to achieve persistence?

Reply:
we wanted to utilize volumeclaimtemplate, it makes handling persistence on multiple replicas easy and was easier to write/maintain as well